### DISCUSSION TITLE
Problema com API Node.js ou Postman

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1781] [Perguntas e Respostas]
Olá Devs, tudo belezinha ?

Galera estou com um proble com um projeto que estou desenvolvendo.  Criei uma  API em Node.js e estou utlizando o Postgres para o banco de dados junto com kenex, na primera fase do projeto  seria de  cadastrar um usuario no banco de dados utilizando o Postman, fiz um validador de formulario em JS e quando vou tentar cadastrar um usuraio no banco de dados não está passando na primeira validação sendo que estou passando de forma correta.

O problema é que verifiquei o codigo está certo, 
pelo que pesquisei pode ser alguma confirguração no Postman mais não encontro nada concreto. Vou estar linkando meu repositorio do projeto, se alguem souber pq não está passando na validação agradeço muito.

Link do Repostorio: https://github.com/Al3xFlores/Projeto-final-Cod3r

**Comment by tcelestino** [comment]
@Al3xFlores, o Postman retorna algum mensagem na resposta?
